### ISSUE TITLE
Better phone number filtering for twillio and SMS in integrations page

### ISSUE THREAD

**Issue by cheeseblubber** [papercups-io/papercups#816]
**Problem**
Right now if you enter a phone number without the area code or have the wrong formatting in phone number section it will fail silently and not work. We should make sure that there is a country code and that it can work regardless of the text formatting

<img width="534" alt="Screen Shot 2021-05-05 at 5 30 52 PM" src="https://user-images.githubusercontent.com/4218509/117211941-a697de80-adc7-11eb-8968-fd87d98e70c9.png">

**Solution**
It should validate that it has a country code and we shouldn't care about formatting

**Comment by cheeseblubber** [comment]
@a8t you might have some context on this

**Comment by a8t** [comment]
Twilio has a free phone number validator service that we already use for something else. Maybe we can also pair with a client-side validation. (is a phone number regex stupid?)

**Comment by cheeseblubber** [comment]
Yeah I think just some client side validation + string formatting would be ideal. It doesn't work if we add spaces in to the phone number
